code/modules/mob/living/simple_animal/friendly/mule_bot/mule_bot.dm

```diff
@@ -0,0 +1,139 @@
+
+/obj/effect/mule_link/Initialize()
+	. = ..()
+	var/mob/living/simple_animal/mule_bot/M = new(loc)
+	var/obj/item/remote/R = new(loc)
+	new /obj/item/mule_module/storage(loc)
+	M.try_link(R)
+	R.bot = M
+	qdel(src)
+
+
+/mob/living/simple_animal/mule_bot
+	name = "Felidae Beetle MK 1"
+	desc = "A highly spohisticated load carrying robotic companion for the advanced marine. Manly as hell!"
+	icon = 'icons/mob/mule_bot.dmi'
+	icon_state = "mule_bot"
+	icon_living = "mule_bot"
+	icon_dead = "mule_bot"
+	gender = FEMALE
+	speak = list("Meow!", "Purr!", "Remember to stock up on medicine!", "Ill carry for the team!", "I sure do hope all these munations are safe...",
+	"Did we really need to bring THAT many plasma cutters?", "Oh I may have the thing for that!","I don't think a xenomorphs head would fit...","Sometimes I wonder if moving bullets is the best way to make a living..","Please stop calling me a tin can...","Time to restock!")
+	speak_emote = list("purrs", "meows")
+	emote_hear = list("meows.", "mews.")
+	emote_see = list("shakes its head.", "shivers.")
+	speak_chance = 5
+	turns_per_move = 5
+	see_in_dark = 6
+	flags_pass = PASSTABLE
+	mob_size = MOB_SIZE_SMALL
+	response_help  = "pets"
+	response_disarm = "gently pushes aside"
+	response_harm   = "kicks"
+	a_intent = INTENT_HELP
+	var/obj/item/mule_module/installed_module
+	var/obj/item/clothing/head/hat
+	var/mutable_appearance/hat_overlay
+
+/mob/living/simple_animal/mule_bot/Initialize()
+	. = ..()
```

Comment:
What's the point of defining this if it's just going to be the same as parent?

Reply:
its leftover code, im just keeping it there for now incase I need it. but your entirly right